### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by davmak** [bluevisiontec/GoogleShoppingApi#46]
I have updated your extension and now I got this error message when I want to manage items : 

Fatal error: require_once(): Failed opening required '/var/www/vhosts/mysite.com/httpdocs/vendor/google/apiclient/src/Google/autoload.php' (include_path='/var/www/vhosts/mysite.com/httpdocs/app/code/local:/var/www/vhosts/mysite.com/httpdocs/app/code/community:/var/www/vhosts/mysite.com/httpdocs/app/code/core:/var/www/vhosts/mysite.com/httpdocs/lib:.:/opt/plesk/php/5.5/share/pear') in /var/www/vhosts/mysite.com/httpdocs/app/code/community/BlueVisionTec/GoogleShoppingApi/Model/GoogleShopping.php on line 2

Before, it worked very well. Any Idea ?

**Comment by schnere** [comment]
Did you update Google API Client for PHP to version 1.1.4 and apply the following patch?

```
--- vendor/google/apiclient/src/Google/Client.php.orig  2015-06-10 12:15:08.937391432 +0200
+++ vendor/google/apiclient/src/Google/Client.php       2015-06-10 12:15:47.693520083 +0200
@@ -16,7 +16,7 @@
  */

 if (!class_exists('Google_Client')) {
-  require_once dirname(__FILE__) . '/../autoload.php';
+  require_once dirname(__FILE__) . '/autoload.php';
 }

 /**
```

**Comment by davmak** [comment]
I saw this information "Please not that version 1.1.4 has an error which will be fixed in 1.1.5" so I used 1.1.2 version.

**Comment by davmak** [comment]
Thanks it works !
